extensions/amp-user-notification/amp-user-notification.md

```diff
@@ -0,0 +1,101 @@
+<!--
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### <a name="amp-user-notification"></a> `amp-user-notification`
+
+Displays a notification that allows the visitor of a site to give consent
```

Comment:
Displays a dismissable notification to the user.

We don't need to document the consent part here at all. That is only one use case anyway. We can document that on the consumer side of the service exposed by this.